sched/scheduler/job_state.go

```diff
@@ -141,15 +141,12 @@ func (j *jobState) taskCompleted(taskId string, running bool) {
 }
 
 // Update JobState to reflect that an error has occurred running this Task
-func (j *jobState) errorRunningTask(taskId string, err error, preempted bool) {
+func (j *jobState) errorRunningTask(taskId string, err error) {
```

Comment:
I believe this was a recent addition of mine that is meant to handle the case where worker goes offline/online and we abort the old run. So I think we still need this but you can verify if that sounds suspect.

Reply:
See other comment, I may have missed an edge case in my last fix, and we may still need this.